File: 3768-check-if-digits-are-equal-in-string-after-operations-i/check-if-digits-are-equal-in-string-after-operations-i.py
class Solution(object):
    def hasSameDigits(self, s):
        """
        :type s: str
        :rtype: bool
        """
        while len(s)>2:
            st=""
            for i in range(1,len(s)):
                m=(int(s[i-1])+int(s[i]))%10
                st+=str(m)
            s=st
        return s[0]==s[-1]

File: 3768-check-if-digits-are-equal-in-string-after-operations-i/check-if-digits-are-equal-in-string-after-operations-i.cs
public class Solution {
    public bool HasSameDigits(string s) {
        while(s.Length>2)
        {
            var st = new StringBuilder();
            for(int i=1; i < s.Length; i++)
            {
                int n = (s[i-1]-'0' + s[i]-'0')%10;
                st.Append(n);
            }
            s=st.ToString();
        }
        return s[0]==s[s.Length-1];
    }
}